src/Range.jsx

```diff
@@ -123,13 +123,10 @@ class Range extends React.Component {
 
     const nextBounds = [...state.bounds];
     nextBounds[state.handle] = value;
-    let nextHandle = state.handle;
+    const nextHandle = state.handle;
     if (props.pushable !== false) {
       const originalValue = state.bounds[nextHandle];
       this.pushSurroundingHandles(nextBounds, nextHandle, originalValue);
-    } else if (props.allowCross) {
```

Comment:
I think no.
the codes about `allowCross` feature is here [https://github.com/react-component/slider/blob/master/src/Range.jsx#L270](https://github.com/react-component/slider/blob/master/src/Range.jsx#L270)
where I delete is update handle controller, isn't it ..